common/cards/alter-egos-ii/hermits/iskallman-rare.ts

```diff
@@ -145,10 +147,17 @@ class IskallmanRareHermitCard extends HermitCard {
 			})
 			attack.addNewAttack(backlashAttack)
 
+			const attackerInfo = HERMIT_CARDS[attacker.row.hermitCard.cardId]
 			const hermitInfo = HERMIT_CARDS[pickedRow.hermitCard.cardId]
 			if (hermitInfo) {
 				const maxHealth = Math.max(pickedRow.health, hermitInfo.health)
 				pickedRow.health = Math.min(pickedRow.health + 50, maxHealth)
+				game.battleLog.addCustomEntry(
```

Comment:
As this is used in so many places now, maybe it's a good idea to rename it to simply "addEntry"?

Reply:
Sure